examples/gitserver/gitserver.yaml

```diff
@@ -70,12 +70,17 @@ items:
           - name: HOOK_PATH
             value: /var/lib/git-hooks
 
-          # The script to use to do custom language detection on a 
+          # If 'yes' new-app will be invoked on push for repositories
+          # for which a matching BuildConfig is not found.
+          - name: GENERATE_ARTIFACTS
+            value: "yes"
```

Comment:
still bothers me we are using "yes" for this when we use "true" everywhere else i can think of.